### ISSUE TITLE
wrong linkedin id link set in the footer

### ISSUE THREAD

**Issue by medss19** [recodehive/awesome-github-profiles#1326]
![image](https://github.com/user-attachments/assets/ff91f12f-a3ff-4787-b7b7-0e87d9b1f285)

when you click on this icon, we are not being redirected to your linkedin id. 

![image](https://github.com/user-attachments/assets/69494973-e6b4-4f4e-8772-08e81c58adde)

`footer-linkedin-icon` needs to be removed.

**Comment by Bishal-35** [comment]
I would like to work on it under SWoC'25. Please assign me.

**Comment by medss19** [comment]
> I would like to work on it under SWoC'25. Please assign me. 

I have already been assigned.

**Comment by Bishal-35** [comment]
Yes @medss19 you have also been assigned another issue. So if I could work on it, it would be helpful.
